woodwork/table_accessor.py

```diff
@@ -674,20 +681,35 @@ def drop(self, columns):
         if not_present:
             raise ColumnNotPresentError(not_present)
 
-        return self._get_subset_df_with_schema([col for col in self._dataframe.columns if col not in columns])
+        if inplace:
+            if dd and isinstance(self._dataframe, dd.DataFrame):
```

Comment:
Thoughts on moving these checks to `_get_subset_df_with_schema`? Because that method uses drop it also won't allow inplace operations for dask and koalas, and that way if we decide to use it for inplace operations elsewhere, the logic is already there.

Reply:
I agree. Fixed!